staging/src/k8s.io/sample-apiserver/pkg/apiserver/apiserver.go

```diff
@@ -32,6 +32,7 @@ import (
 	flunderstorage "k8s.io/sample-apiserver/pkg/registry/wardle/flunder"
 )
 
+// Scheme - defines a new runtime scheme
```

Comment:
please remove the `-` from all the comments.
them comments them self seem fine.

Reply:
All `-` have been removed.